### ISSUE TITLE
Front End Issues

### ISSUE THREAD

**Issue by Deepak-Sangle** [ujwaljp/CS253_Project_MindHive#9]
The webapp is not completely responsive. One can see a lot of issue when working on some mobile phones or Tablet.
For. eg. The tags section is fixed and disturbing with the questions section.

**Comment by ris27hav** [comment]
That's right. Actually, none of us had any web dev experience before implementing this project. So, we managed to present a bare minimum of our app (a little more than that :) ). We are definitely planning to improve UI and make the web pages more responsive in future updates.
